activiti-services/activiti-services-query/activiti-services-query-rest/src/main/java/org/activiti/services/query/events/TaskAssignedEvent.java

```diff
@@ -16,30 +16,22 @@
 
 package org.activiti.services.query.events;
 
-import org.activiti.services.query.model.Task;
+import org.activiti.services.query.es.model.TaskES;
 
 public class TaskAssignedEvent extends AbstractProcessEngineEvent {
 
-    private Task task;
+	private TaskES task;
```

Comment:
@gmalanga not sure to understand why we need this in the normal query-rest service? Are we mixing stuff here? 
I will be happy with having a shared model or two separate models between ES and JPA, but not a mixed one. How is TaskES different from Task and why we replaced Task for TaskES in the JPA option?